### PR TITLE
tuxedo-keyboard: 3.0.8 -> 3.0.9

### LINKED ISSUE #154483: tuxedo-keyboard module isn't loaded after install
### Describe the bug

After adding `linuxPackages.tuxedo-keyboard` to `/etc/nixos/configuration.nix` and restarting the kernel modules aren't loaded.

### Steps To Reproduce
1. Add `linuxPackages.tuxedo-keyboard` to `/etc/nixos/configuration.nix`
2. nixos-rebuild switch
3. reboot

### Expected behavior

I should be able to see the installed modules in `lsmod`, however none are loaded.

### Screenshots

\-

### Additional context

```sh
$ uname -a 
Linux nixos 5.10.88 #1-NixOS SMP Wed Dec 22 08:31:00 UTC 2021 x86_64 GNU/Linux

$ nix-store --query --requisites /run/current-system | cut -d- -f2- | sort | uniq | grep tuxedo
tuxedo-keyboard-5.10.88-3.0.8

$ nix-store -q --references /run/current-system/sw | grep tuxedo
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8

$ find /nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib/modules
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib/modules/5.10.88
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib/modules/5.10.88/clevo_acpi.ko
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib/modules/5.10.88/clevo_wmi.ko
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib/modules/5.10.88/tuxedo_io.ko
/nix/store/hqpj454zzics2vvbywsg3x9w2270wzpp-tuxedo-keyboard-5.10.88-3.0.8/lib/modules/5.10.88/tuxedo_keyboard.ko

$ sudo modprobe tuxedo_keyboard
modprobe: FATAL: Module tuxedo_keyboard not found in directory /run/booted-system/kernel-modules/lib/modules/5.10.88
```

### Notify maintainers

@blanky0230 (@blitz ?)

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 5.10.88, NixOS, 21.11 (Porcupine)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.16`
 - channels(root): `"nixos-21.11.334814.04bd2d1a470"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

### PR DESCRIPTION
**Pull request by blitz**
###### Motivation for this change

Updates the TUXEDO keyboard driver to the latest version. While I was here, I added user requested documentation (#154483).

Closes #154483.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/os-specific/linux/tuxedo-keyboard/default.nix
Comment (blanky0230) [inline pkgs/os-specific/linux/tuxedo-keyboard/default.nix:32]:
```suggestion
Can be used with the "hardware.tuxedo-keyboard" NixOS module.
```

Since strictly speaking the module and package distinct things and users could always just add 

```
      boot.kernelModules = ["tuxedo_keyboard"];
      boot.extraModulePackages = [ boot.kernelPackages.tuxedo-keyboard ];
```

**Review by blanky0230 — APPROVED** [review]
Otherwise lgtm. I can't merge though someone with merge access should have a look as well :)

**Comment by lheckemann** [comment]
Merged in 1291b29

**Comment by blitz** [comment]
@lheckemann Thanks for fixing the comment and driving this over the finishing line. :+1:

**Pull request closed without merging**
